addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -77,6 +77,8 @@ def _check_move_configuration(self, move):
             error_message.append(_("- Transportation Mode"))
         elif move.l10n_in_mode == "0" and not move.l10n_in_transporter_id:
             error_message.append(_("- Transporter is required when E-waybill is managed by transporter"))
+        elif move.l10n_in_mode == "0" and move.l10n_in_transporter_id and not move.l10n_in_transporter_id.vat:
+            error_message.append(_("- Selected Transporter is missing GSTIN?"))
```

Comment:
```suggestion
            error_message.append(_("- Selected Transporter is missing GSTIN"))
```

Reply:
Also, we should add it in translation files :slightly_smiling_face:

Reply:
I made some changes to the pull request based on your feedback.
Can you re-check this?

Thank you